components/gc-featured-link/reports/a11y-2-fr.html

```diff
@@ -0,0 +1,16 @@
+---
+{
+	"title": "Assessment de l'accessibilité #1 - Lien en vedette",
+	"language": "fr",
+	"description": "Évaluationvaluation des critères de succès aux niveaux A et AA du WCAG 2.1",
+	"tag": "gc-featured-link",
+	"parentdir": "gc-featured-link",
+	"parentPage": "Lien en vedette",
+	"parentPageURL": "gc-featured-link",
```

Comment:
We don't need all those following front-matter variable for a page in GCWeb because they are not use.

```suggestion
```

Reply:
It is back, can you remove it.

Reply:
done